webapp/templates/team/partials/submission_list.html.twig

```diff
@@ -52,7 +52,16 @@
                 <td>
                     <a data-ajax-modal data-ajax-modal-after="markSeen" {% if link %}href="{{ link }}"{% endif %}>
                         {%- if submission.submittime > submission.contest.endtime %}
-                            {{ 'too-late' | printResult }}
+                            {{ 'too-late' | printResult }} 
+                            {%- if showTooLateResult %}
+                                ({%- if submission.judgings.first is empty or submission.judgings.first.result is empty %}
+                                    {{- '' | printResult -}}
+                                {%- elseif verificationRequired and not submission.judgings.first.verified %}
+                                    {{- '' | printResult -}}
+                                {%- else %}
+                                    {{- submission.judgings.first.result | printResult -}}
+                                {%- endif %})
+                            {%- endif %}
```

Comment:
Is there a way to remove this duplication?

Reply:
I've thought of this, but didn't find a better way to implement this.

The following code also deplicate some code but is shorter. Maybe it's better?
```
{%- if submission.submittime > submission.contest.endtime %}
    {{ 'too-late' | printResult }} 
{%- endif %}
{%- if submission.submittime <= submission.contest.endtime or showTooLateResult %}
    {%- if submission.submittime > submission.contest.endtime %}
        (
    {%- endif %}
    {%- if submission.judgings.first is empty or submission.judgings.first.result is empty %}
        {{- '' | printResult -}}
    {%- elseif verificationRequired and not submission.judgings.first.verified %}
        {{- '' | printResult -}}
    {%- else %}
        {{- submission.judgings.first.result | printResult -}}
    {%- endif %}
    {%- if submission.submittime > submission.contest.endtime -%}
        )
    {%- endif %}
{%- endif %}
```

Reply:
I like it a bit more. If you accept `too-late/<verdict>` you can also remove the last check.

Reply:
Okay, I've made a commit of this.